src/main/java/com/jalasoft/selenium/Ariel/movies/Customer.java

```diff
@@ -0,0 +1,91 @@
+package com.jalasoft.selenium.Ariel.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * Created by Ariel Mattos on 3/6/2017.
+ */
+
+class Customer {
+    private String name;
+    private Vector rentals = new Vector();
```

Comment:
Change to use a List instead of a Vector. Iteration was changed to use a for loop.